src/pages/Integrations/Create/IntegrationsStep.tsx

```diff
@@ -2,18 +2,25 @@ import {
   componentTypes,
   validatorTypes,
 } from '@data-driven-forms/react-form-renderer';
-import {
-  compileAllIntegrationComboOptions,
-  iconMapper,
-  nextDetailsStep,
-} from './helpers';
+import { compileAllIntegrationComboOptions, iconMapper } from './helpers';
 import { CARD_SELECT } from './CreateWizard';
 import { defaultIconList } from '../../../config/Config';
 
 export const integrationsStep = (category: string) => ({
   title: 'Select Integration type',
-  name: 'integrationType',
-  nextStep: ({ values }) => nextDetailsStep(values),
+  name: 'integration-type',
+  nextStep: {
+    when: 'integration-type',
+    stepMapper: {
+      'camel:slack': 'slack-details',
+      'camel:google_chat': 'teams-gchat-details',
+      'camel:teams': 'teams-gchat-details',
+      'camel:splunk': 'splunk-details',
+      'camel:servicenow': 'service-now-details',
```

Comment:
Please import the `IntegrationType` and use for instance `[IntegrationType.SLACK]: 'slack-details'` that way if we were to change the type we don't have to change it here as well.